app/components/primer/octicon_component.rb

```diff
@@ -33,6 +33,13 @@ class OcticonComponent < Primer::Component
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(icon_name = nil, icon: nil, size: SIZE_DEFAULT, use_symbol: false, **system_arguments)
       icon_key = icon_name || icon
+
+      # Don't allow sizes under 16px
+      if system_arguments[:height].present? && system_arguments[:height] < 16 || system_arguments[:width].present? && system_arguments[:width] < 16
+        system_arguments.delete(:height)
+        system_arguments.delete(:width)
```

Comment:
I wonder if we should accept those arguments at all.
maybe we could only accept `size` and that'd be either one of the accepted symbols or an int >= 16

Reply:
I think we should deprecated the use of these, but there's a lot of cleanup work to do on dotcom to prepare to only also `size:` there's a lot of violations that would break if we forced their hand. The most visible of which would be the github logo in the header.